src/Mantasflowers.WebApi/Validation/Order/PostOrderContactDetailsRequestValidator.cs

```diff
@@ -8,7 +8,7 @@ public class PostOrderContactDetailsRequestValidator : AbstractValidator<PostOrd
         public PostOrderContactDetailsRequestValidator()
         {
             RuleFor(x => x.Email)
-                .NotEmpty();
+                .NotNull();
```

Comment:
Could you remind again what this is about? Orders without email or what?

Reply:
Yeah, depends if we will allow creating order without Email (it is in requirements, tho).